docs/pages/reference/cli.mdx

```diff
@@ -387,6 +387,113 @@ $ tsh play --format=pty 1fe153d1-ce8b-4ef4-9908-6539457ba4ad.tar
 $ tsh play --format=json ~/play/0c0b81ed-91a9-4a2a-8d7c-7495891a6ca0.tar | jq '.event
 ```
 
+### tsh proxy db
+
+Start a local TLS proxy for database connections when using Teleport with TLS
+Routing enabled. Clients can connect to a Teleport-registered database through
+the local proxy.
+
+```code
+$ tsh proxy db [<flags>] <db>
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--cert-file` | none | string | Path to the certificate file for configuring TLS on the local proxy.|
+| `--cluster` | none | string | The name of the Teleport cluster to connect to. |
+| `--key-file` | none | string | Path to the private key file for configuring TLS on the local proxy.|
+| `--db-name` | none | string | Optional database name to log in to. |
+| `--db-user` | none | string | Optional database user to log in as. |
+| `--port` | string | Source port used by the local proxy.|
```

Comment:
This line is missing the default value column. I assume is should also be `none`?